src/js/media-control-bar.js

```diff
@@ -31,7 +31,8 @@ template.innerHTML = `
     }
 
     media-progress-range,
-    ::slotted(media-progress-range) {
+    ::slotted(media-progress-range),
```

Comment:
We might want to create a formal class here, like ".media-timeline", to make it easier to make an element expand to full width. Not important for this PR.